cargo-paradedb/Cargo.toml

```diff
@@ -18,20 +17,28 @@ glob = "0.3.1"
 itertools = "0.12.1"
 once_cell = "1.19.0"
 reqwest = { version = "0.12.7", features = ["json", "blocking"] }
+ureq = { version = "2.6", features = ["json"] }
 serde = "1.0.210"
 serde_json = "1.0.128"
 sqlx = { version = "0.7.4", features = [
   "postgres",
   "runtime-async-std",
+  "uuid",
   "chrono",
   "tls-native-tls",
 ] }
 tempfile = "3.12.0"
 tracing = "0.1.40"
-tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
-
-[dev-dependencies]
-rstest = "0.18.2"
-
-[package.metadata.cargo-machete]
-ignored = ["chrono"]
```

Comment:
Why remove this?

Reply:
My Bad—I’ve restored the dependency in the `toml` file.